manual/cypher/cypher-docs/src/docs/dev/ql/merge/index.asciidoc

```diff
@@ -23,6 +23,10 @@ If there are multiple matches, they will all be passed on to later stages of the
 The last part of +MERGE+ is the +ON CREATE+ and +ON MATCH+.
 These allow a query to express additional changes to the properties of a node or relationship, depending on if the element was ++MATCH++ed in the database or if it was ++CREATE++d.
 
+The rule planner expands a `MERGE` pattern from the end point that has the identifier with the lowest lexicographical order.
```

Comment:
We can't assume the reader knows about planners, so I'd suggest changing this sentence like this:
```The rule planner (see <<how-are-queries-executed>>) expands a `MERGE` pattern from the end point that has the identifier with the lowest lexicographical order.```